app/views/facility_journals/show.html.haml

```diff
@@ -16,62 +16,59 @@
       = f.input :status_string
   .span4
     .well
-      - if Settings.financial.journal_format.xls
-        %p=link_to 'Download XLS File', @journal.file.url
-      - if Settings.financial.journal_format.xml
-        %p=link_to 'Download XML File', facility_journal_path(current_facility, @journal, :format => 'xml')
-      - if Settings.financial.journal_format.csv
-        %p=link_to 'Download CSV File', facility_journal_path(current_facility, @journal, :format => 'csv')
+      - %i(xls xml csv).each do |format|
```

Comment:
I think we also need this on `facility_journals/index` if there's a pending journal.